smartsim/launcher/step/lsfStep.py

```diff
@@ -123,6 +123,22 @@ def __init__(self, name, cwd, run_settings):
         if not self.run_settings.in_batch:
             self._set_alloc()
 
+
+    def get_output_files(self):
+        """Return two paths to error and output files based on cwd"""
+        output = self.get_step_file(ending=".out")
+        error = self.get_step_file(ending=".err")
+
+        if self.run_settings.individual_suffix:
```

Comment:
lets do some explaining here in comments. be verbose because this doesn't make any sense

Reply:
OK